src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -72,6 +73,8 @@ const UserProfileModal: React.FunctionComponent<PropTypes> = ({
 
   const usersById = { [userProfile.id]: userProfile };
 
+  const hasBadges = isTruthy(venue?.showBadges);
```

Comment:
Maybe keep this as `showBadges`?

`hasBadges` implies to me that we've checked an array of badges and it's not empty/similar, whereas this just seems to be whether the venue config is turned on or not.